blocks/colors/style.scss

```diff
@@ -0,0 +1,87 @@
+p.has-pale-pink-background-color {
```

Comment:
Are these styles meant to be limited to `p` elements?

Reply:
Sorry I updated the code but missed the answer. During the first version, I expected the styles to be per theme and the class to applied always to the most external dom element. So themes would need to specify how to apply the color to different blocks. But the overhead would be huge. It is better if we just set the rule according to its "context" (background, color) and then blocks are responsible to apply the class to any dom element they have.